galaxy_ng/app/dynaconf_hooks.py

```diff
@@ -738,3 +739,17 @@ def configure_dab_required_settings(settings: Dynaconf) -> Dict[str, Any]:
         if key.isupper() and settings.get(key, notset) is notset:
             data[key] = getattr(dynamic_settings, key)
     return data
+
+
+def configure_dab_rbac_settings(settings: Dynaconf) -> Dict[str, Any]:
+    data = {}
+    if not hasattr(settings, 'ANSIBLE_BASE_MANAGED_ROLE_REGISTRY'):
+        data['ANSIBLE_BASE_MANAGED_ROLE_REGISTRY'] = {}
+    if not hasattr(settings, 'ANSIBLE_BASE_ALLOW_SINGLETON_ROLES_API'):
+        data['ANSIBLE_BASE_ALLOW_SINGLETON_ROLES_API'] = True
+    if not hasattr(settings, 'ANSIBLE_BASE_ALLOW_SINGLETON_USER_ROLES'):
+        data['ANSIBLE_BASE_ALLOW_SINGLETON_USER_ROLES'] = False
+    if not hasattr(settings, 'ANSIBLE_BASE_ALLOW_SINGLETON_TEAM_ROLES'):
+        data['ANSIBLE_BASE_ALLOW_SINGLETON_TEAM_ROLES'] = False
+    data["INSTALLED_APPS"] = settings.INSTALLED_APPS[:] + ["ansible_base.rbac"]
```

Comment:
Using a unique merge will prevent duplication if this app is for some reason already in the list of installed apps.
```suggestion
    data["INSTALLED_APPS"] = ["ansible_base.rbac", "dynaconf_merge_unique"]
```